### ISSUE TITLE
Can this be used in nodejs?

### ISSUE THREAD

**Issue by RudolfVonKrugstein** [fiduswriter/diffDOM#113]
Hi, firstly thank you for the great project!

I am trying to use diffDOM in nodejs.
When I do:

```
const diffDOM = require("diff-dom")
...
const dd = new diffDOM.DIffDOM();
let a = diffDOM.stringToObj(html1);
let b = diffDOM.stringToObj(html2);
dd.diff(a, b);
```

I get:
```
ReferenceError: HTMLElement is not defined
```

Is this usable in NodeJS or does this only work in the browser?
Thank you!

**Comment by johanneswilm** [comment]
Hey, @RudolfVonKrugstein Generally yes, this should be runable in just nodejs. I am guessing that others ran it in combination with jsdom, so they did have a global HTMLElement defined. My commit should fix it also for your usecase. I will look at another issue and then make a release.

**Comment by johanneswilm** [comment]
@RudolfVonKrugstein  This has been released in version 4.2.7.
